spacy_experimental/biaffine_parser/pytorch_bilinear.py

```diff
@@ -16,7 +61,7 @@ def __init__(
 
         self.head = nn.Linear(nI, hidden_width)
         self.dependent = nn.Linear(nI, hidden_width)
-        self.bilinear = nn.Bilinear(hidden_width, hidden_width, nO)
+        self.bilinear = Bilinear(hidden_width, nO)
```

Comment:
We should add a comment mentioning the performance issue with torch's implementation (either here or in the module definition).